tests/connections/test_unix_socket_connection.py

```diff
@@ -68,37 +69,60 @@ def tearDown(self):
         os.unlink(self.socketname)
 
     def test_unix_socket_connection_connect_read(self):
-        self.connection = UnixSocketConnection(
+        connection = UnixSocketConnection(
             path=self.socketname, timeout=DEFAULT_TIMEOUT
         )
-        self.connection.connect()
-        self.connection.read()
-        self.connection.disconnect()
+        connection.connect()
+        connection.read()
+        connection.disconnect()
 
     def test_unix_socket_connection_connect_send_bytes_read(self):
-        self.connection = UnixSocketConnection(
+        connection = UnixSocketConnection(
             path=self.socketname, timeout=DEFAULT_TIMEOUT
         )
-        self.connection.connect()
-        self.connection.send(bytes("<gmp/>", 'utf-8'))
-        self.connection.read()
-        self.connection.disconnect()
+        connection.connect()
+        connection.send(bytes("<gmp/>", 'utf-8'))
+        connection.read()
+        connection.disconnect()
 
     def test_unix_socket_connection_connect_send_str_read(self):
-        self.connection = UnixSocketConnection(
+        connection = UnixSocketConnection(
             path=self.socketname, timeout=DEFAULT_TIMEOUT
         )
-        self.connection.connect()
-        self.connection.send("<gmp/>")
-        self.connection.read()
-        self.connection.disconnect()
+        connection.connect()
+        connection.send("<gmp/>")
+        connection.read()
+        connection.disconnect()
```

Comment:
It seems that all these already existing unix socket tests also don't ensure the behavior by asserting on something. I am really not sure if such tests are useful. IMHO they only test if no exception is raised.

Reply:
I will look at them again!